pyaedt/desktop.py

```diff
@@ -181,9 +181,11 @@ def release_desktop(close_projects=True, close_desktop=True):
                 desktop.CloseProject(project)
         pid = _main.oDesktop.GetProcessID()
         if settings.aedt_version >= "2022.2" and settings.use_grpc_api and not is_ironpython:
-            import ScriptEnv
+            import PyDesktopPlugin
 
-            ScriptEnv.Release()
+            PyDesktopPlugin.Release()
+            _delete_objects()
```

Comment:
@maxcapodi78 Maybe here we could add a comment to explain why we are releasing this way. It might be not trivial for external contributors.

Reply:
correct. Thanks